src/api/public/apidocs-new/paths/source_project_name_package_name_cmd_release.yaml

```diff
@@ -0,0 +1,68 @@
+post:
+  summary: Release sources and binaries of a specified package.
+  description: |
+    Release sources and binaries of the specified package. If no release target is specified through
+    the optional parameters, it will trigger the release for all manual set release targets
+    set on the specified project repository definitions for the specified package.
+  security:
+    - basic_authentication: []
+  parameters:
+    - $ref: '../components/parameters/project_name.yaml'
+    - $ref: '../components/parameters/package_name.yaml'
+    - in: query
+      name: target_project
+      schema:
+        type: string
+      description: |
+        The name of the release target project. Specify in which project the sources and
+        binaries are going to be released. You need to provide the `target_repository`
+        parameter as well.
+      example: OBS:Server:2.10
+    - in: query
+      name: target_repository
+      schema:
+        type: string
+      description: |
+        The name of the release target repository. Specify in which repository the sources and
+        binaries are going to be released. You need to provide the `target_project`
+        parameter as well.
+      example: openSUSE_Tumbleweed
+    - in: query
+      name: repository
+      schema:
+        type: string
+      description: |
+        Limit the release to a certain repository, set on the project repository definitions.
+      example: 'openSUSE_Tumbleweed'
+    - in: query
+      name: setrelease
+      schema:
+        type: string
+      description: |
+        If this parameter is present, the release will be tagged with this parameter's value.
+      example: Build8.18
+  responses:
+    '200':
+      $ref: '../components/responses/succeeded.yaml'
+    '403':
```

Comment:
I miss an example of a 400 error. We should list all the possible errors and provide at least an example. Do you have at hand an example this error? Maybe you haven't faced any of this error in your tests.